src/test/scala/edu/berkeley/ce/rockslicing/EndToEndSpec.scala

```diff
@@ -11,11 +11,11 @@ class EndToEndSpec extends FunSuite {
   test("Simple end-to-end test using unit cube and simple planes") {
     // Read input file to generate list of joints and initial rock block
     val inputSource = Source.fromURL(getClass.getResource(s"/$INPUT_FILE_NAME"))
-    val (rockVolume, jointList) = InputProcessor.readInput(inputSource)
+    val (rockVolume, jointList) = InputProcessor.readInput(inputSource).get
     inputSource.close()
 
     // Create an initial block
-    val blocks = Vector(Block((0.0, 0.0, 0.0), rockVolume))
+    val blocks = Seq(Block((0.0, 0.0, 0.0), rockVolume))
```

Comment:
This will need to be updated when `InputProcessor.scala` is changed to read the global origin coordinates.